paramnormal/dist.py

```diff
@@ -808,6 +808,88 @@ def fit(cls, data, **guesses):
         return cls.param_template(loc=params[0], lambda_=params[1]**-1)
 
 
+class rice(BaseDist_Mixin):
+    """
+    Create and fit data to a Rice distribution.
+
+    Methods
+    -------
+    fit
+        Use scipy's maximum likelihood estimation methods to estimate
+        the parameters of the data's distribution. By default, `loc`
+        is fixed at 0. Thus, only `R` and `sigma` are estimated unless
+        `loc` is explicitly set to `None`.
+    from_params
+        Create a new distribution instances from the ``namedtuple``
+        result of the :meth:`~fit` method.
+
+    Parameters
+    ----------
+    R : float
+        The shape parameter of the distribution.
+    sigma : float
+        The standard deviate of the distribution.
+    loc : float, optional
+        Location parameter of the distribution. This defaults to, and
+        should probably be left at, 0.
+
+        .. note ::
+           When fitting an Rice distribution to a dataset, this
+           will be fixed at its default value unless you explicitly set
+           it to other values. Set to `None` if you wish that it be
+           estimated entirely from scratch.
+
+    Examples
+    --------
+    >>> import numpy
+    >>> import paramnormal as pn
+    >>> numpy.random.seed(0)
+    >>> pn.rice(R=10, sigma=2).rvs(size=3)
+    array([ 15.67835764,  13.36907874,  10.37753817])
+
+    >>> # you can also use greek letters
+    >>> numpy.random.seed(0)
+    >>> pn.rice(R=10, σ=2).rvs(size=3)
+    array([ 15.67835764,  13.36907874,  10.37753817])
+
+    >>> # silly fake data
+    >>> numpy.random.seed(0)
+    >>> data = pn.rice(R=10, sigma=2).rvs(size=37)
+    >>> # pretend `data` is unknown and we want to fit a dist. to it
+    >>> pn.rice.fit(data)
+    params(R=10.100674084593422, sigma=1.759817171541185, loc=0)
+
+    >>> # include `loc` in the estimate (bad idea)
+    >>> pn.rice.fit(data, loc=None)
+    params(R=4.249154300734, sigma=1.862167512728, loc=5.570921659394)
+
+    References
+    ----------
+    http://scipy.github.io/devdocs/generated/scipy.stats.rice
+    https://en.wikipedia.org/wiki/Rice_distribution
+
+    See Also
+    --------
+    scipy.stats.rice
+    numpy.random.exponential
+
+    """
+    dist = stats.rice
+    param_template = namedtuple('params', ['R', 'sigma', 'loc'])
+    name = 'rice'
+
+    @staticmethod
+    @utils.greco_deco
+    def _process_args(R=None, loc=0, sigma=None, fit=False):
+        loc_key, scale_key = utils._get_loc_scale_keys(fit=fit)
+        return {loc_key: loc, scale_key: sigma, 'b': R/sigma if R is not None else None}
```

Comment:
OK: I have removed two comments.  I think you are correct.  Consider the first item in the "related distributions" section of the Wikipedia article and let theta be zero.  Then nu is the mean value of X, and X is a normal RV with standard deviation sigma.  To make the scipy Rice match this, we do need to divide R by sigma before feeding it in as the shape parameter:
```
In [9]: ss.rice(100/10, scale=10).rvs(10)
array([ 109.25263972,  110.09863048,   86.55780539,  103.11784999,
         86.5479606 ,  105.39289188,  119.8328802 ,   93.23215964,
        108.60585299,  107.74609675])
```
So indeed, the scipy version is worse than I remembered; it is not only that the second parameter is the useless `loc`, making it too easy to accidentally set the `loc` when intending to set the `scale` (standard deviation, `sigma`, `nu`, etc.), but even the first parameter is scaled in a non-standard way.  What a confusing mess!  Really, your paramnormal needs to be incorporated into scipy, and the present underlying interface needs to be withdrawn to the background as a low-level implementation detail.